modules/draco/src/lib/draco-module-loader.js

```diff
@@ -89,7 +72,7 @@ function initializeDracoDecoder(DracoDecoderModule, wasmBinary) {
 // ENCODER
 
 async function loadDracoEncoder(options) {
-  let DracoEncoderModule = await loadLibrary('draco_encoder.js', 'draco', options);
+  let DracoEncoderModule = await loadLibrary(DRACO_ENCODER_URL, 'draco', options);
```

Comment:
Where does this load from? How does this file get installed?

Reply:
It's not final. I updated with your branch and latest master

Reply:
OK Looks like this one still has to be loaded locally, it is not available on CDN.